modular_skyrat/modules/customization/game/objects/items/tanks/n2_tanks.dm

```diff
@@ -4,7 +4,7 @@
 
 /obj/item/tank/internals/nitrogen
 	name = "nitrogen tank"
-	desc = "A tank of nitrogen. Designed specifically for Vox"
+	desc = "A tank of nitrogen. Designed specifically for Vox."
```

Comment:
```suggestion
	desc = "A small tank of nitrogen, for crew who don't breathe the standard air mix."
```
Since Nitrogen lungs aren't specifically a Vox thing any more it'd probably be better to not name-drop them